src/js/components/purchaseOrder/PurchaseOrderListHeader.jsx

```diff
@@ -16,10 +18,10 @@ const PurchaseOrderListHeader = ({ history, supportedActivities }) => (
       <Button
         defaultLabel="Create Shipment from PO"
         label="react.purchaseOrder.createShipmentFromPo.label"
-        onClick={() => history.push('/openboxes/stockMovement/createCombinedShipments?direction=INBOUND')}
+        onClick={() => history.push({ pathname: STOCK_MOVEMENT_URL.createCombinedShipments(), search: 'direction=INBOUND' })}
```

Comment:
wouldn't it be possible for link to accept eventual search params instead? I guess it would be cleaner, but on the other hand I don't mind this version.